h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -413,7 +413,7 @@ final public M trainModelNested(Frame fr) {
     if (error_count() > 0)
       throw H2OModelBuilderIllegalArgumentException.makeFromBuilder(this);
     startClock();
-    if( !nFoldCV() ) trainModelImpl().compute2();
+    if( !nFoldCV() ) H2O.submitTask(trainModelImpl()).join();
```

Comment:
good catch! Just curious why it was implemented the way it was: by calling `compute2` directly, was it not completely ignoring the priority queue?

Reply:
IMHO this was done in error, not intentionally. I don't really see the purpose and since we have this working this way in the CV case anyway, I think we are fine here.